monkey/tests/unit_tests/monkey_island/cc/repositories/test_local_storage_file_repository.py

```diff
@@ -4,14 +4,19 @@
 from unittest.mock import Mock, patch
 
 import pytest
+from monkeytoolbox import get_os
+from monkeytypes import OperatingSystem
 from tests.monkey_island.utils import assert_linux_permissions, assert_windows_permissions
 from tests.utils import raise_
 
-from common.utils.environment import is_windows_os
 from monkey_island.cc import repositories
 from monkey_island.cc.repositories import LocalStorageFileRepository
 
 
+def is_windows_os():
```

Comment:
The calls for this function could be replaced with the boolean condition. Either that or:

```suggestion
def os_is_windows():
```

See https://guardicore.atlassian.net/wiki/spaces/RES/pages/2800353435/Infection+Monkey+Development+Guidelines#Functions/Methods-returning-booleans